packages/frontend/src/components/UserSettings/UserAttributesPanel/index.tsx

```diff
@@ -0,0 +1,243 @@
+import { subject } from '@casl/ability';
+import { OrgAttribute } from '@lightdash/common';
+import {
+    Button,
+    Flex,
+    Group,
+    Modal,
+    Stack,
+    Table,
+    Text,
+    Title,
+    Tooltip,
+} from '@mantine/core';
+import {
+    IconAlertCircle,
+    IconCircleX,
+    IconEdit,
+    IconInfoCircle,
+} from '@tabler/icons-react';
+import { FC, useState } from 'react';
+import { useOrganization } from '../../../hooks/organization/useOrganization';
+import { useTableStyles } from '../../../hooks/styles/useTableStyles';
+import {
+    useUserAttributes,
+    useUserAttributesDeleteMutation,
+} from '../../../hooks/useUserAttributes';
+import { useApp } from '../../../providers/AppProvider';
+import LoadingState from '../../common/LoadingState';
+import MantineIcon from '../../common/MantineIcon';
+import { SettingsCard } from '../../common/Settings/SettingsCard';
+import ForbiddenPanel from '../../ForbiddenPanel';
+import UserAttributeModal from './UserAttributeModal';
+
+const UserListItem: FC<{
+    orgUserAttribute: OrgAttribute;
+    allUserAttributes: OrgAttribute[];
+}> = ({ orgUserAttribute, allUserAttributes }) => {
+    const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
+    const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
+
+    const { mutate: deleteUserAttribute } = useUserAttributesDeleteMutation();
+
+    return (
+        <>
+            <tr>
+                <td width={800}>
```

Comment:
(note for self - check this in preview env)

Reply:
instead of having one TD and placing both text and action items in one cell you can create a second cell with `width="1%"` << this is a "hack" to set cell with minimal width.
that way you can get rid of `width={800}` and next line of `<Flex justify="space-between" align="center">`